clients/ui/frontend/src/style/MUI-theme.scss

```diff
@@ -237,21 +242,24 @@
 }
 
 .pf-v6-c-form-control input {
+  // TODO: https://issues.redhat.com/browse/RHOAIENG-14702,
```

Comment:
Small nit, we shouldn't check Red Hat references into the community codebase, could you remove the link to the internal JIRA please? :)